vdr/vdr.go

```diff
@@ -91,7 +91,21 @@ func (r *VDR) Configure(_ core.ServerConfig) error {
 
 // Diagnostics returns the diagnostics for this engine
 func (r *VDR) Diagnostics() []core.DiagnosticResult {
-	return []core.DiagnosticResult{}
+	// return # conflicted docs
+	count := 0
+	r.store.Iterate(func(doc did.Document, metadata types.DocumentMetadata) error {
+		if metadata.IsConflicted() {
```

Comment:
It looks like this is going to be expensive when there are many documents?

Reply:
I'll add something to #109, since you'll require an index and search functionality. It's better to do this when storage changes.